src/main/java/com/spaceclub/event/service/ParticipationService.java

```diff
@@ -56,6 +60,19 @@ public void apply(EventParticipationCreateInfo info) {
         eventUserRepository.save(newEventUser);
     }
 
+    private void eventAlreadyParticipated(EventParticipationCreateInfo info, Optional<EventUser> optionalEventUser) {
+        if (optionalEventUser.isPresent()) {
+            EventUser eventUser = optionalEventUser.get();
+
```

Comment:
isPresent로 존재 유무 확인후 get으로 꺼내는것보다 ifPresent로 바로 꺼내서 쓰면 더 코드가 깔끔해질 것 같아요
```suggestion
        optionalEventUser.ifPresent(
            (eventUser) -> ...
        )
```

Reply:
[chore: 리뷰 반영](https://github.com/Space-Club/Backend/pull/215/commits/f3cb9ea4a0165608d0f3e866d108e6f72fa81229)